platform/windows/os_windows.cpp

```diff
@@ -139,13 +139,13 @@ void RedirectIOToConsole() {
 
 	if (AttachConsole(ATTACH_PARENT_PROCESS)) {
 		// Restore redirection (Note: if not redirected it's NULL handles not INVALID_HANDLE_VALUE).
-		if (h_stdin != nullptr) {
+		if (h_stdin != INVALID_HANDLE_VALUE) {
 			SetStdHandle(STD_INPUT_HANDLE, h_stdin);
 		}
-		if (h_stdout != nullptr) {
+		if (h_stdout != INVALID_HANDLE_VALUE) {
 			SetStdHandle(STD_OUTPUT_HANDLE, h_stdout);
 		}
-		if (h_stderr != nullptr) {
+		if (h_stderr != INVALID_HANDLE_VALUE) {
```

Comment:
> If the function fails, the return value is INVALID_HANDLE_VALUE. To get extended error information, call GetLastError.

> If an application does not have associated standard handles, such as a service running on an interactive desktop, and has not redirected them, the return value is NULL.

Not sure way this can fail, but in this case checking both `INVALID_HANDLE_VALUE` and `NULL` might be more correct, but we definitely expect `NULL`.